pkg/operator/k8sutil/pod.go

```diff
@@ -181,7 +181,19 @@ func DeleteDeployment(clientset kubernetes.Interface, namespace, name string) er
 		_, err := clientset.ExtensionsV1beta1().Deployments(namespace).Get(name, metav1.GetOptions{})
 		return err
 	}
-	return deletePodsAndWait(namespace, name, deleteAction, getAction)
+	return deleteResourceAndWait(namespace, name, "deployment", deleteAction, getAction)
+}
+
+// DeleteDeployment makes a best effort at deleting a deployment and its pods, then waits for them to be deleted
```

Comment:
The comment function name is not matching

Reply:
good catch